packages/edit-site/src/components/page-pages/index.js

```diff
@@ -131,12 +144,35 @@ export default function PagePages() {
 						</a>
 					);
 				},
+				elements: [
```

Comment:
The `elements` defined by a field could be used in different scenarios: filters, when editing the value of the cell, etc.